dev/com.ibm.websphere.appserver.features/visibility/public/cloudant-1.0/resources/l10n/com.ibm.websphere.appserver.cloudant-1.0.properties

```
@@ -16,6 +16,8 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables connections to Cloudant databases by providing a connector instance that's configured \
- in the server configuration and can be injected or accessed through Java Naming and Directory Interface (JNDI). Applications use the \
+description=This feature has been stabilized. \
```

While acceptable to just remove this, it might be nice to direct the customers in the right direction like Mongo does with:
The strategic alternative to this feature is to create a CDI producer by following the technique that is described at https://www.ibm.com/links?url=https%3A%2F%2Fopenliberty.io%2Fblog%2F2019%2F02%2F19%2Fmongodb-with-open-liberty.html.

---

```suggestion
description=This feature has been stabilized. \
The strategic alternative to this feature is to create a CDI producer by following the technique as described in https://openliberty.io/blog/2019/02/19/mongodb-with-open-liberty.html \
```
nitpick: the link Tracy provided was a redirect link.  I think posting the original link is preferable and the knowledge center site will handle the re-direct where necessary.